src/components/taskDetails/index.tsx

```diff
@@ -218,6 +220,13 @@ const TaskDetails: FC<Props> = ({ url, taskID }) => {
                         </section>
 
                         <section className={classNames.rightContainer}>
+                            <button
+                                onClick={() =>
+                                    router.push(`/progress/${taskID}?dev=true`)
```

Comment:
take this out as a function

Reply:
Function just for 1 line?

Reply:
Since it uses Router I need to use router.push inside a component.